docs/http_api.rst

```diff
@@ -165,6 +165,66 @@ Channels
    Delete a channel.
 
 
+Logs
+^^^^
+
+.. http:get:: /channels/(channel_id:str)/logs
+
+   Get the most recent logs for a specific channel.
+
+   :query int n:
+       The amount of logs to fetch.
+
+
+   The response is a list of logs, with each log taking the following form:
+
+   :param str logger: The logger that created the log, usually the channel id.
+   :param int level:
+       The level of the logs. Corresponds to the levels found in the python
+       module :py:mod:`logging`.
+   :param float timestamp: Timestamp, in the format of seconds since the epoch.
+   :param str message: The message of the log.
+
+   In the case of an exception, the following fields may also be returned:
+
+   :param str class: The class of the exception.
+   :param str instance: The specific instance of the exception.
+   :param list traceback:
+       A list of strings representing the traceback of the error.
+
+   **Example Request**:
+
+   .. sourcecode:: http
+
+       GET /channels/123-456-7a90/logs?n=2 HTTP/1.1
+       Host: example.com
+       Accept: application/json, text/javascript
+
+   **Example response**:
+
+   .. sourcecode:: json
+
+      {
+        status: 200,
+        code: "OK",
+        description: "Logs retrieved",
+        result: [
+            {
+                logger: "123-456-7a90",
+                level: 20,
+                timestamp: 987654321.0,
+                message: "Last log for the channel"
+            },
+            {
+                logger: "123-456-7a90",
+                level: 20,
+                timestamp: 987654320.0,
+                message: "Second last log for the channel"
+            }
```

Comment:
I added an example in https://github.com/praekelt/junebug/commit/d3cf551eae900f4c6036a50e5330a594d2617606